karabo_bridge/client.py

```diff
@@ -46,12 +45,10 @@ class Client:
     ------
     NotImplementedError
         if socker type or serialization algorythm is not supported.
-    SyntaxError
+    ZMQError
         if provided endpoint is not valid.
     """
     def __init__(self, endpoint, sock='REQ', ser='msgpack'):
-        if re.match(r'^tcp://.*:\d{1,5}$', endpoint) is None:
-            raise SyntaxError("Provided endpoint is invalid:", str(endpoint))
```

Comment:
ZMQ already does its own check. I also don't like using SyntaxError, because that means that Python code is invalid.